### ISSUE TITLE
Add th cells to Activity Log

### ISSUE THREAD

**Issue by KimberlyMunoz** [cfpb/consumerfinance.gov#977]
The activity log should have visually hidden `th` cells to help screenreaders navigate the data table.

https://github.com/cfpb/cfgov-refresh/pull/972/files#r39313740

**Comment by GrooveCS** [comment]
Is this still an Open Issue?

**Comment by Scotchester** [comment]
No.
